File: LevelEntities/OilService.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace LevelEntities
{
    public class OilService
    {
            public int ID { get; set; }
            public int CustomerID { get; set; }
            public String CustomerPlate { get; set; }
            public String CustomerName { get; set; }
            public String CustomerPhone { get; set; }
            public String Grade { get; set; }
            public int Miles { get; set; }
            public String OilType { get; set; }
            public int ChangeMiles { get; set; }
            public String TodayDate { get; set; }
            public String ChangeDate { get; set; }

        public OilService() { }

            public OilService(int ID, int CustomerID, String CustomerPlate, String CustomerName, String CustomerPhone, String Grade, int Miles, String OilType, int ChangeMiles, String TodayDate, String ChangeDate)
            {
                this.ID = ID;
                this.CustomerID = CustomerID;
                this.CustomerPlate = CustomerPlate;
                this.CustomerName = CustomerName;
                this.CustomerName = CustomerPhone;
                this.Grade = Grade;
                this.Miles = Miles;
                this.OilType = OilType;
                this.ChangeMiles = ChangeMiles;
                this.TodayDate = TodayDate;
                this.ChangeDate = ChangeDate;
            }
        }
    }

File: LevelPresentation/js/oilservice.js
function templateRow() {
    var template = "<tr>";
    template += ("<td>" + "" + "</td>");
    template += ("<td>" + "" + "</td>");
    template += ("<td>" + "" + "</td>");
    template += ("<td>" + "" + "</td>");
    template += ("<td>" + "" + "</td>");
    template += ("<td>" + "" + "</td>");
    template += ("<td>" + "" + "</td>");
    template += "</tr>";
    return template;
}

function addRow() {
    var template = templateRow();
    for (var i = 0; i < 10; i++) {
        $("#tbl_body_table").append(template);
    }
}

//Add data to DataTable tbl_oilservices
function addRowDT(data) {
    var tabla = $("#tbl_oilservices").DataTable();
    for (var i = 0; i < data.length; i++) {
        tabla.fnAddData([
            data[i].CustomerPlate,
            data[i].CustomerName,
            data[i].CustomerPhone,
            data[i].Grade,
            data[i].Miles,
            data[i].OilType,
            data[i].TodayDate,
            data[i].ChangeDate
        ]);
    }
}


//Send Data to DataTable in ListOilService Page with Ajax
function sendDataAjax() {
    $.ajax({
        type: "POST",
        url: "frmListOilService.aspx/ListOilService",
        data: {},
        contentType: 'application/json; charset=utf-8',
        error: function (xhr, ajaxOptions, thrownError) {
            console.log(xhr.status + " \n" + xhr.responseText, "\n" + thrownError);
        },
        success: function (data) {
            console.log(data.d);
            //addRowDT(data.d);
        }
    });
}


//Calling to SendDataAjax Function Javacsript
sendDataAjax(); 